src_c/scrap.c

```diff
@@ -27,6 +27,8 @@
 
 #include "SDL_syswm.h"
 
+#include "SDL_clipboard.h"
+
```

Comment:
Is this needed? SDL.h is included somewhere down the tree of pygame.h, and SDL.h seems like it would include SDL_clipboard.h.